src/main/java/com/byznass/tiolktrack/postgres/dao/PostgresUserProvider.java

```diff
@@ -43,6 +43,21 @@ public User getUser(String userId) throws NoUserWithSuchIdException {
 
 	}
 
+	@Override
+	public boolean exists(String userId) throws TiolkTrackException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)